src/main/java/com/emt/model/internal/EloRatingChange.java

```diff
@@ -0,0 +1,10 @@
+package com.emt.model.internal;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import java.math.BigDecimal;
+
+@Embeddable
+public record EloRatingChange(
+    @Column(name = "winner_rating_change") BigDecimal winnerRatingChange,
+    @Column(name = "loser_rating_change") BigDecimal loserRatingChange) {}
```

Comment:
Unnecessary complication. There's no reason to use an embedded class, as both BigDecimal always represent the same number, just negated for the loser